go.mod

```diff
@@ -12,11 +12,11 @@ require (
 	github.com/checkpoint-restore/checkpointctl v1.2.1
 	github.com/checkpoint-restore/go-criu/v7 v7.1.0
 	github.com/containernetworking/plugins v1.5.1
-	github.com/containers/buildah v1.37.0
-	github.com/containers/common v0.60.1-0.20240806133927-05b2e1fd7cd0
+	github.com/containers/buildah v1.37.1
+	github.com/containers/common v0.60.1
```

Comment:
This is piratically speaking a downgrade as we were already vendoring c/common from main,
I Still have a open common vendor in https://github.com/containers/podman/pull/23519 that also shows the you most likely need the test fix from there assuming the "breaking" c/image change was back ported in 5.32.1

Reply:
So next steps?  Just back this out and vendor directly into the v5.2 branch?

Reply:
I would vendor c/common@main here at least.
But without the test fix nothing going to pass so you either cherry-pick that commit or (what I prefer) merge https://github.com/containers/podman/pull/23519 first and then rebase this.

Reply:
I would wait for the 5.2 PR at least until all the test fixes are review/merged at least.

Reply:
I've added the test fix for now, off to a meeting, will see add the c/common bump here.  I'll talk to @mheon shortly to see if makes sense to cherry-pick this, or just vendor Buildah in the v5.2 branch and add the test fix.

Thanks!